### DISCUSSION TITLE
What version of Ubuntu is currently used?

### DISCUSSION THREAD

**Discussion by sebastienros** [TechEmpower/FrameworkBenchmarks#7218] [General]
We (MS) recently upgraded our machines to Ubuntu 20.04 and noticed some benchmarks were impacted (security patches, ...). Also 22.04 is around the corner.

**Comment by joanhey** [comment]
Ubuntu 20.04.3 LTS from 31 Jan
https://github.com/TechEmpower/FrameworkBenchmarks/issues/6591#issuecomment-1026041557

And waiting for the update to 22.04 when released.
The new kernel in 22.04 fix a lot of performance degradation, from the Spectre / Meltdown Mitigation security fixes. That we noted from round 18 to 19.
https://www.phoronix.com/scan.php?page=news_item&px=FSGSBASE-For-Linux-5.9
